react/src/lib/components/SubsurfaceViewer/storybook/SubsurfaceViewer.stories.jsx

```diff
@@ -306,6 +305,37 @@ const meshMapLayer = {
     colorMapName: "Physics",
 };
 
+const meshMapLayerPng = {
+    "@@type": "MapLayer",
```

Comment:
I think we should only use this method when we specifically want to demonstrate de-serialization from Json. Suggest using `new` instantiation instead.